lib/matplotlib/axes/_axes.py

```diff
@@ -2292,6 +2292,14 @@ def bar(self, x, height, width=0.8, bottom=None, *, align="center",
         xerr = kwargs.pop('xerr', None)
         yerr = kwargs.pop('yerr', None)
         error_kw = kwargs.pop('error_kw', {})
+        ezorder = error_kw.pop('zorder', None)
+        if ezorder is None:
+            ezorder = kwargs.pop('zorder', None)
```

Comment:
This pops the `zorder` out of the kwargs.  Do we want to use `kwargs.get('zorder', None)`?